packages/blade/src/utils/index.ts

```diff
@@ -0,0 +1 @@
+export { default as useColorScheme } from './useColorScheme';
```

Comment:
Why exporting from utils? Are we going to do such re-exports everywhere?

Reply:
yes